packages/contracts/contracts/optimistic-ethereum/OVM/execution/OVM_ExecutionManager.sol

```diff
@@ -1817,20 +2013,23 @@ contract OVM_ExecutionManager is iOVM_ExecutionManager, Lib_AddressResolver {
     )
         internal
     {
-        // Avoid unnecessary the SSTORE.
+        // These conditionals allow us to avoid unneccessary SSTOREs.  However, they do mean that the current storage
+        // value for the messageContext MUST equal the _prevMessageContext argument, or an SSTORE might be erroneously skipped.
```

Comment:
Interesting comment. Was this the result of something you found while working on this PR?